static/js/util/profile_edit.js

```diff
@@ -50,7 +50,7 @@ export function boundSelectField(keySet, label, options) {
   } = this.props;
   let onChange = value => {
     let clone = _.cloneDeep(profile);
-    _.set(clone, keySet, value? value.value : null);
+    _.set(clone, keySet, value? value.value : '');
```

Comment:
When I was trying to click save and continue with an empty value, it would show the error but then immediately reload.
Also, in my other PR, George mentioned something about react complaining about null values.